cmd/internal/cli/actions_linux.go

```diff
@@ -427,7 +433,7 @@ func execStarter(cobraCmd *cobra.Command, image string, args []string, name stri
 
 	// convert image file to sandbox if image contains
 	// a squashfs filesystem
-	if UserNamespace && fs.IsFile(image) {
+	if (UserNamespace || parentUserNs) && fs.IsFile(image) {
```

Comment:
the variable `UserNamespace` means a user namespace is going to be set up.

The comment above this line needs updating to reflect the conditions that are being checked in order to enter this branch.